homeassistant/components/comelit/alarm_control_panel.py

```diff
@@ -0,0 +1,148 @@
+"""Support for Comelit VEDO system."""
+from __future__ import annotations
+
+import logging
+
+from aiocomelit.api import ComelitVedoAreaObject
+from aiocomelit.const import ALARM_AREAS, AlarmAreaState
+
+from homeassistant.components.alarm_control_panel import (
+    AlarmControlPanelEntity,
+    AlarmControlPanelEntityFeature,
+    CodeFormat,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import (
+    STATE_ALARM_ARMED_AWAY,
+    STATE_ALARM_ARMED_HOME,
+    STATE_ALARM_ARMED_NIGHT,
+    STATE_ALARM_ARMING,
+    STATE_ALARM_DISARMED,
+    STATE_ALARM_DISARMING,
+    STATE_ALARM_TRIGGERED,
+)
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.typing import StateType
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import DOMAIN
+from .coordinator import ComelitVedoSystem
+
+_LOGGER = logging.getLogger(__name__)
+
+ALARM_ACTIONS: dict[str, str] = {
+    "disable": "dis",  # Disarm
+    "home": "p1",  # Arm P1
+    "night": "p12",  # Arm P1+P2
+    "away": "tot",  # Arm P1+P2 + IR / volumetric
+}
+ALARM_AREA_ARMED_STATUS: dict[str, int] = {
+    "home_p1": 1,
+    "home_p2": 2,
+    "night": 3,
+    "away": 4,
```

Comment:
Can we use constants for this?

Reply:
I don't get what you looking for.

Reply:
I don't know what `home_p1` means. Something like `AREA_ARMED_STATUS_HOME` or something like that would be clearer to read.